### DISCUSSION TITLE
web support in Attachment Preview

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5868] [Feature Requests]
It just shows the file name at the moment.
<img width="1501" alt="Screenshot 2023-08-31 at 3 12 45 PM" src="https://github.com/specify/specify7/assets/53784701/1fd288f2-4b8c-4297-b834-36b9d6093735">

**Comment by maxpatiiuk** [comment]
Whether image displays or not is most likely dependent on your browser.
Which browser did you use?
Although, all of them should support the WebP format by now - https://caniuse.com/webp

**Comment by alesan99** [comment]
WebP images are still not displaying for me either (Windows 11 Chrome). Dragging the file directly onto a separate Chrome tab displays it correctly, so presumably they are supported (?).
